packages/material-ui/src/Dialog/Dialog.test.js

```diff
@@ -181,6 +181,46 @@ describe('<Dialog />', () => {
         'should not fire the onBackdropClick callback',
       );
     });
+
+    it(
+      'should not close if the dialog receives the mousedown and the ' +
+        'backdrop receives the mouseup',
+      () => {
+        const onBackdropClick = spy();
+        wrapper.setProps({ onBackdropClick });
+
+        const backdrop = wrapper.find('div');
```

Comment:
```suggestion
        const backdrop = wrapper.find('[role="document"]');
```
Following a11y best practices helps testing.